index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/75d86a86f1.js" crossorigin="anonymous"></script>
+        <script src="script.js" defer></script>
+        <link rel="stylesheet" href="styles.css">
+        <title>Youth Empowering Africa Summit </title>
+        </head>
+    <body>
+        <header class="header">
+          <section class="top">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a href="#">Log out</a>
+            <a href="#">My page</a>
+            </section>
+            <nav class="navigation-section">
+                <i class="fa-sharp fa-solid fa-bars"></i>
+                <a href="index.html"><img src="images/YCA Main Logo.png" class ="home-logo" alt="logo"></a>   
+                  <ul class="nav-items">
+                      <i class="fa-solid fa-x"></i>
+                      <li><a href="about.html" class="about-us-link nav-link">About</a></li>
+                      <li><a href="" class="disabled nav-link">Program</a></li>
+                      <li><a href="" class="disabled nav-link">Join</a></li>
+                      <li><a href="" class="disabled nav-link">Sponsors</a></li>
+                      <li><a href="" class="disabled nav-link">News</a></li>
+                      </ul>                
+            </nav>
```

Comment:
- [ ] While the content of the page is to be personalized, the design should still look very much like that of Cindy, in this case, the content is not at the extreme ends of the left and right, there are spaces, in fact, the right space is under the logout of the top short dark bar. Kindly make your design follow that also.

## Your design

![image](https://user-images.githubusercontent.com/86200356/233288468-b62dd370-aec8-496a-babb-0d8e23353176.png)

## Behance design

![image](https://user-images.githubusercontent.com/86200356/233288643-5bcd2c1d-db85-44b3-ac69-fa37c1c377e5.png)

Reply:
Fixed.